semcore/tooltip/src/Tooltip.jsx

```diff
@@ -137,7 +139,103 @@ function TooltipPopper(props) {
   );
 }
 
-const Tooltip = createComponent(
+class HintRoot extends TooltipRoot {
+  static displayName = 'Hint';
+  static style = style;
+  static enhance = [uniqueIDEnhancement(), resolveColorEnhance()];
+  static defaultProps = {
+    ...defaultProps,
+    liveRegion: false,
+  };
+  subcomponents = [Hint.Trigger.displayName, Hint.Popper.displayName];
+  defaultChildren = (title, Children, props) => (
+    <>
+      <Hint.Trigger {...props}>
+        <Children />
+      </Hint.Trigger>
+      <Hint.Popper>{title}</Hint.Popper>
+    </>
+  );
+
+  getTriggerProps() {
+    const props = super.getTriggerProps();
+    return {
+      ...props,
+      'aria-describedby': undefined,
+      'aria-label': this.asProps.title,
+    };
+  }
+
+  getPopperProps() {
+    const props = super.getPopperProps();
+    return {
+      ...props,
+      'aria-hidden': true,
+      role: undefined,
+      children: this.asProps.title,
+    };
+  }
+}
+
+class DescriptionTooltipRoot extends TooltipRoot {
+  static displayName = 'DescriptionTooltip';
+  static style = style;
+  static enhance = [uniqueIDEnhancement(), resolveColorEnhance()];
+  static defaultProps = {
+    ...defaultProps,
+    liveRegion: false,
+    interaction: 'click',
+  };
+  popperRef = React.createRef();
+  subcomponents = [DescriptionTooltip.Trigger.displayName, DescriptionTooltip.Popper.displayName];
+  defaultChildren = (title, Children, props) => (
+    <>
+      <Hint.Trigger {...props}>
+        <Children />
+      </Hint.Trigger>
+      <Hint.Popper>{title}</Hint.Popper>
+    </>
+  );
+  handlePopperVisibleChange = (visible) => {
+    if (visible) {
+      setTimeout(() => {
+        this.popperRef.current.focus();
+      }, 0);
+    }
```

Comment:
why not in `uncontrolledProps` ?

Reply:
Cause I don't want to additionally store  `visible` in the DescriptionTooltip. It doesn't work perfect and seems a to be risky a bit